core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -605,6 +607,84 @@ public FileSystemMasterView getFileSystemMasterView() {
   }
 
   /**
+   * Checks the consistency of the subtree under the path.
+   *
+   * @param path the subtree root to check
```

Comment:
nit: change to "the root of the subtree to check" - we check the subtree not the root